about.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<html>
+  <body>
+    <h1>Hello, world!</h1>
```

Comment:
Update the contents of the h1 tag to be something more relevant to an about page... like "All About Me!" instead of "Hello, world!"

There is nothing actually "wrong" with your code, but I am suggesting changes on your PR so you know what it looks like.